mplc/contributivity.py

```diff
@@ -156,7 +154,7 @@ def compute_SV(self, the_scenario):
         characteristic_function = []
 
         for coalition in coalitions:
-            characteristic_function.append(self.not_twice_characteristic(coalition, the_scenario))
+            characteristic_function.append(self.not_twice_characteristic(coalition, self.scenario))
```

Comment:
Not sure to follow here why `not_twice_characteristic` is taking a second arg whereas the method expect only one?
`def not_twice_characteristic(self, subset)`. Am I missing something?

Reply:
No you are right I missed two calls to this function while refacting